svelte/src/components/markdown/markdown.svelte

```diff
@@ -28,8 +28,10 @@
   // TODO: support gitlab too
   const changeSrcIfNeeded = (element: HTMLElement | Element) => {
     const src = element.getAttribute("src");
-    if (src?.startsWith("/")) {
-      element.setAttribute("src", `https://raw.githubusercontent.com${src}`);
+    const isRelativePath = src?.match(/^[^/]+\/[^/].*$|^\/[^/].*$/);
```

Comment:
just curious, it would be simpler to `!src.startsWith('http')` right?

Reply:
I think for our purposes that will be fine. Of course there are many other non http protocols/prefixes that can be used in absolute paths but for the sake of sniffing them out in README’s checking for http should do just fine :)